tracer/test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -669,6 +676,48 @@ private void HandlePotentialDataStreams(MockHttpParser.MockHttpRequest request)
             }
         }
 
+        private void HandleEvpProxyPayload(MockHttpParser.MockHttpRequest request)
+        {
+            if (ShouldDeserializeTraces && request.ContentLength >= 1)
+            {
+                try
+                {
+                    var body = ReadStreamBody(request);
+                    if (request.Headers.GetValue("Content-Encoding") == "gzip")
```

Comment:
Do you use gzipping too? I didn't see anything about that? If not, can probably remove this bit for simplicity

Reply:
True, done in https://github.com/DataDog/dd-trace-dotnet/pull/3322/commits/e65ccbfc53a8a19b691e32b998542e4a8b174ba0